content/es/404.md

```diff
@@ -0,0 +1,2 @@
+## 404: No se pudo encontrar la página Page could not be found
+### ENOENT: No existe el archivo o directorio
```

Comment:
Uhm, this was just a little node'ish pun I added (this is the error message you get if you try to load a non-existing file via `fs.readFile`). Not sure if we want to keep/translate this. ;)